values/jobs/certs-aws.gotmpl

```diff
@@ -7,11 +7,11 @@ tasks:
   certs-aws:
     type: job
     description: Import certs into ACM
-    enabled: {{ and (eq $v.cluster.provider "aws") $v.otomi.hasCloudLB }}
+    enabled: true
     image:
-      repository: {{ $c | get "jobs.certs-aws.image.repository" "otomi/otomi-tasks" }}
-      tag: {{ $c | get "jobs.certs-aws.image.tag" "v0.2.1" }}
-      pullPolicy: {{ $c | get "jobs.certs-aws.image.pullPolicy" "IfNotPresent" }}
+      repository: otomi/otomi-tasks
```

Comment:
I like this change but I am really curious what has changed your mind recently. I remember you were fighting for making all jobs configurable....